forge-gui/res/cardsfolder/upcoming/analyze_the_pollen.txt

```diff
@@ -0,0 +1,8 @@
+Name:Analyze the Pollen
+ManaCost:G
+Types:Sorcery
+S:Mode$ OptionalCost | EffectZone$ All | ValidCard$ Card.Self | ValidSA$ Spell | Cost$ CollectEvidence<8> | Description$ As an additional cost to cast this spell, you may collect evidence 8. (Exile cards with total mana value 8 or greater from your graveyard.)
+A:SP$ ChangeZone | Origin$ Library | Destination$ Hand | Condition$ OptionalCost | ConditionOptionalPaid$ False | ChangeType$ Land.Basic | ChangeNum$ 1 | ConditionCheckSVar$ X | ConditionSVarCompare$ EQ0 | SubAbility$ DBChangeZone | SpellDescription$ Search your library for a basic land card. If evidence was collected, instead search your library for a creature or land card. Reveal that card, put it into your hand, then shuffle.
```

Comment:
for the condition better use the paidList

Reply:
any pointers ?

Reply:
_Crimestopper Sprite_ does it correctly

Reply:
does it work like this? you shouldn't need the CastSA part because there's no reason to switch context

Reply:
I think so yeah 
![Uploading image.png…]()

Reply:
you should still get rid of it